crates/driver/src/infra/config/file/mod.rs

```diff
@@ -557,6 +560,9 @@ enum UniswapV3Config {
 
         /// The URL used to connect to uniswap v3 subgraph client.
         graph_url: Url,
+
```

Comment:
It might be required here as well
```suggestion
        /// How often the liquidity source should be reinitialized to get
        /// access to new pools.
```